### ISSUE TITLE
Checkboxes are inaccessible to VoiceOver

### ISSUE THREAD

**Issue by Flameborn** [AdguardTeam/AdGuardForSafari#283]
VoiceOver, MacOS's built-in screen reader is not able to detect and interact with checkboxes in the main AdGuard for Safari App.

This is quite often because the checkbox is hidden by the UI library, only exposing the outer **div** element, which is not marked up via ARIA tags, e.g. **role**, in this case as a checkbox. As a result, VoiceOver can see the checkbox label, however, there is no indication that this item can be checked/unchecked, like a regular checkbox.

## To reproduce

1. Launch VoiceOver via **cmd+f5** Use **ctrl+option+arrows** to navigate, **ctrl+option+shift+up or down** to stop or start interacting (i.e. access child elements). **ctrl+option+space** activates an item.
2. Navigate to any checkbox. VoiceOver should announce the label and that the control type is a checkbox. Activating it should toggle between states.
3. Use **cmd+f5** again to turn VoiceOver off.
